octavia_f5/controller/worker/l2_sync_manager.py

```diff
@@ -27,8 +27,7 @@
 from octavia.network import base
 from octavia.network import data_models as network_models
 from octavia_f5.common import constants
-from octavia_f5.controller.worker.flows import f5_flows
-from octavia_f5.controller.worker.tasks import f5_tasks
+from octavia_f5.controller.worker.flows import f5_flows, f5_flows_rseries
```

Comment:
If you have a specific name for a module that is working with R-series devices, you should have also specific name for old one (something like `f5_flows_bigip` instead of `f5_flows`)